src/index.html

```diff
@@ -22,6 +22,349 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
```

Comment:
The `header` tag already represents a header, so wrapping it with a `div` with a class `header` is redundant and can be removed for cleaner and more semantic HTML.